run_all_tests.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sbt clean compile coverage test it:test scalastyle coverageReport
```

Comment:
Can you add dependencyUpdates to this please.
Also update all the dependencies with up arrows from [here](https://catalogue.tax.service.gov.uk/repositories/interest-restriction-return)